src/pages/SchoolsMapPage.tsx

```diff
@@ -139,23 +139,18 @@ const SchoolsMapPage: FC<RouteComponentProps> = () => {
   const schools = data?.allSchools;
 
   const bounds: LatLngBoundsLiteral | undefined = useMemo(() => {
-    let b: LatLngBoundsLiteral = [];
+    const b: LatLngBoundsLiteral = [];
```

Comment:
I'd prefer for this line to be as close to `forEach` as possible so that it forms a cohesive block for reading